### PR TITLE
added nodejs12 and removed nodejs8

### LINKED ISSUE #46: add nodejs 12, remove nodejs8

### PR DESCRIPTION
**Pull request by gurrrung**
closes #46

### CODE REVIEW

**Comment by gurrrung** [comment]
kotlin and golang are failing and I feel they don't have anything to do with this PR.

**Comment by gurrrung** [comment]
Ready for review.

File: tests/judge-worker/nodejs12/script.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-const readline = require('readline');
+var readline = require('readline');
```
Comment (championswimmer) [inline tests/judge-worker/nodejs12/script.js:1]: do not change to var please.

Comment (gurrrung) [reply tests/judge-worker/nodejs12/script.js]: done.

**Comment by gurrrung** [comment]
ping

**Pull request merged**